extension/js/common/settings.ts

```diff
@@ -372,4 +372,31 @@ export class Settings {
     }
   }
 
+  static populateAccountsMenu = async (page: 'index.htm' | 'inbox.htm') => {
+    const menuAcctHtml = (email: string, picture = '/img/svgs/profile-icon.svg', isHeaderRow: boolean) => {
+      return [
+        `<div ${isHeaderRow && 'id = "header-row"'} class="row alt-accounts action_select_account">`,
+        '  <div class="col-sm-10">',
+        `    <div class="row contains_email" data-test="action-switch-to-account">${Xss.escape(email)}</div>`,
+        '  </div>',
+        `  <div><img class="profile-img" src="${Xss.escape(picture)}" alt=""></div>`,
+        '</div>',
+      ].join('');
+    };
+    const acctEmails = await Store.acctEmailsGet();
+    const acctStorages = await Store.getAccounts(acctEmails, ['picture', 'setup_done']);
+    for (const email of acctEmails) {
+      Xss.sanitizePrepend('#alt-accounts', menuAcctHtml(email, acctStorages[email].picture, page === 'inbox.htm'));
+    }
+    $('#alt-accounts img.profile-img').on('error', Ui.event.handle(self => {
+      $(self).off().attr('src', '/img/svgs/profile-icon.svg');
+    }));
+    $('.action_select_account').click(Ui.event.handle(target => {
+      const acctEmail = $(target).find('.contains_email').text();
+      const acctStorage = acctStorages[acctEmail];
+      window.location.href = acctStorage.setup_done
+        ? Env.urlCreate(page, { acctEmail })
```

Comment:
Let me know your style policy on the ternary operator.

Reply:
Currently, if it fits on one line, do `cond ? a : b` on a single line. If it doesn't, do a normal multi-line if/else, including curly brackets.

Reply:
You could shorten it by doing if/else inside:
```js
Env.urlCreate(acctStorage.setup_done ? xx : yy, { acctEmail })
```